test/unittests/commands/xmlupload/make_rdf_graph/test_make_values.py

```diff
@@ -11,49 +10,58 @@
 from dsp_tools.commands.xmlupload.make_rdf_graph.constants import KNORA_API
 from dsp_tools.commands.xmlupload.make_rdf_graph.make_values import _make_one_prop_graph
-from dsp_tools.commands.xmlupload.models.deserialise.deserialise_value import XMLProperty
+from dsp_tools.commands.xmlupload.models.formatted_text_value import FormattedTextValue
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryBoolean
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryColor
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryDate
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryDecimal
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryGeometry
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryGeoname
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryInt
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryInterval
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryLink
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryList
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryRichtext
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediarySimpleText
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryTime
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryUri
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntervalFloats
+from dsp_tools.commands.xmlupload.models.lookup_models import IRILookup
 from dsp_tools.commands.xmlupload.models.lookup_models import JSONLDContext
-from dsp_tools.commands.xmlupload.models.lookup_models import Lookups
 from dsp_tools.commands.xmlupload.models.permission import Permissions
 from dsp_tools.commands.xmlupload.models.permission import PermissionValue
 from dsp_tools.models.exceptions import BaseError
-from dsp_tools.models.exceptions import InputError
-from dsp_tools.models.exceptions import PermissionNotExistsError
-from dsp_tools.models.exceptions import UserError
+from dsp_tools.utils.date_util import Calendar
+from dsp_tools.utils.date_util import Date
+from dsp_tools.utils.date_util import Era
+from dsp_tools.utils.date_util import SingleDate
 
 ONTO = Namespace("http://0.0.0.0:3333/ontology/9999/onto/v2#")
-namespaces = {"onto": ONTO, "knora-api": KNORA_API}
 
 PERMISSION_LITERAL = Literal("CR knora-admin:ProjectAdmin", datatype=XSD.string)
 RES_ONE_URI = URIRef("http://rdfh.ch/9999/res_one")
 
 
-@pytest.fixture
-def permissions_lookup() -> dict[str, Permissions]:
-    return {"open": Permissions({PermissionValue.CR: ["knora-admin:ProjectAdmin"]})}
+OPEN_PERMISSION = Permissions({PermissionValue.CR: ["knora-admin:ProjectAdmin"]})
+
+
+def onto_str(prop: str) -> str:
```

Comment:
The name is not intuitive enough, I suggest `dereference_prop_name()` or something similar.

Reply:
I want it short and simple. It is only in the test code.